src/app/services/evento.service.ts

```diff
@@ -0,0 +1,110 @@
+import { Injectable } from '@angular/core';
+import { environment } from 'src/environments/environment';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Evento } from '../models/evento';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Frontend/pullRequest?prid=7138940)
```suggestion
import { Evento } from "../models/evento";
```